indico/modules/events/contributions/operations.py

```diff
@@ -76,6 +81,11 @@ def update_contribution(contrib, data):
     """
     rv = {'unscheduled': False, 'undo_unschedule': None}
     current_session_block = contrib.session_block
+    start_dt = data.pop('start_date', None)
+    if start_dt is not None:
+        if not contrib.timetable_entry:
```

Comment:
Sorry for that, the two above snippets are remnants from the time I started working on this thing.